src/framework/input/element-input.js

```diff
@@ -369,9 +369,13 @@ class ElementInput {
         this._touchmoveHandler = this._handleTouchMove.bind(this);
         this._sortHandler = this._sortElements.bind(this);
 
+        /** @type {ElementComponent[]} */
```

Comment:
Presumably this appears in the JSDoc output?

Reply:
Thank you for noting, I always forget `@ignore` :see_no_evil: